phpunit/functional/CommonDBTMTest.php

```diff
@@ -1908,4 +1909,65 @@ public function testFilterValues($classname, $add_input, $update_input)
         $this->checkInput($item, $item->getID(), $update_input);
     }
 
+
+    /**
+     * Provider for self::testGetSpecificMassiveActions().
+     */
+    public static function getSpecificMassiveActionsProvider()
+    {
+        /** @var \DBmysql $DB */
+        global $DB;
+        // Test presence of "Add transfer list" action
+        foreach (get_declared_classes() as $class) {
+            if (
+                is_subclass_of($class, \CommonDBTM::class)
+                && !(new \ReflectionClass($class))->isAbstract()
+            ) {
```

Comment:
`get_declared_classes()` may not provide expected results if all classes are not loaded first.

```suggestion
        foreach ($this->getClasses() as $class) {
            if (is_subclass_of($class, \CommonDBTM::class)) {
```